misk-metrics-testing/src/test/kotlin/misk/metrics/FakeMetricsTest.kt

```diff
@@ -101,30 +101,30 @@ class FakeMetricsTest {
 
     histogram.record(450.0)
     assertThat(metrics.histogramP50("call_times")).isEqualTo(400.0)
-    assertThat(metrics.histogramP99("call_times")).isEqualTo(400.0)
-
-    histogram.record(500.0)
-    assertThat(metrics.histogramP50("call_times")).isEqualTo(400.0)
     assertThat(metrics.histogramP99("call_times")).isEqualTo(450.0)
 
-    histogram.record(550.0)
+    histogram.record(500.0)
     assertThat(metrics.histogramP50("call_times")).isEqualTo(450.0)
     assertThat(metrics.histogramP99("call_times")).isEqualTo(500.0)
 
-    histogram.record(600.0)
+    histogram.record(550.0)
     assertThat(metrics.histogramP50("call_times")).isEqualTo(450.0)
     assertThat(metrics.histogramP99("call_times")).isEqualTo(550.0)
+
+    histogram.record(600.0)
+    assertThat(metrics.histogramP50("call_times")).isEqualTo(500.0)
+    assertThat(metrics.histogramP99("call_times")).isEqualTo(600.0)
   }
 
   @Test
   internal fun `get all samples`() {
-    metrics.counter("counter", "-", listOf("foo")).labels("bar").inc()
+    metrics.counter("counter_total", "-", listOf("foo")).labels("bar").inc()
```

Comment:
Technically this line didn't need to change, just the sample name below [see this comment for details](https://github.com/cashapp/misk/pull/2695/files#r1113709699)